completions/xfreerdp

```diff
@@ -1,14 +1,23 @@
 # xfreerdp completion                                      -*- shell-script -*-
 
+_comp_cmd_xfreerdp__kbd_list()
+{
+    local kbd_list
+    # Trying non deprecated /list:kbd first
+    kbd_list=$("$1" /list:kbd 2>/dev/null) ||
+        kbd_list=$("$1" --kbd-list 2>/dev/null) ||
+        kbd_list=$("$1" /kbd-list 2>/dev/null)
```

Comment:
I'm not familiar with xfreerdp, but according to [this page](https://github.com/awakecoding/FreeRDP-Manuals/blob/88c0158c896565214e6b145f67e95efe8dee883f/User/FreeRDP-User-Manual.markdown#syntax), the windows style `/kbd-list` seems to be preferred to the posix-style `--kbd-list`.

Also, `--kbd-list` seems to be a synonym of `/kbd-list`, so maybe do you think we can only test `/kbd-list` and drop `--kbd-list`? Conversely, if we are going to test both the styles, I think we should also check `--list:kbd`.

Reply:
I think we should probably only test stuff that we know works in some places?
`--list:kbd` doesn't seem to work in fedora at least. Regarding only testing the slashes variant, I'm not familiar either, so I don't have a strong opinion, I just tried to keep it consistent with what it did before.

Reply:
I tried it in Fedora 41, Arch, and Ubuntu 20 LTS, but none seem to support the POSIX-style options (i.e., neither `--kbd-list` nor `--list:kbd` / `--list kbd`). Maybe the POSIX-style options are removed at all, and the above page is too old.

Anyway, the POSIX-style options seem to have been discouraged even when they were supported, and now they don't seem to work at all. Even if we would keep the test for `--kbd-list`, I think it should be tested after trying `/kbd-list`.

Reply:
Thanks!